app/helpers/blacklight/render_constraints_helper_behavior.rb

```diff
@@ -47,54 +65,53 @@ def render_constraints_query(localized_params = params)
   #
   # @param [ActionController::Parameters] localized_params query parameters
   # @return [String]
-  def remove_constraint_url(localized_params)
-    scope = localized_params.delete(:route_set) || self
```

Comment:
I /think/ this behavior is unnecessary with the switch to `search_action_path`  (probably mirroring what's going on with the facets anyway).